src/appengine/handlers/cron/triage.py

```diff
@@ -301,6 +411,12 @@ def get(self):
     # been removed.
     all_jobs = data_handler.get_all_job_type_names()
 
+    bug_filed_24_hours_per_job = {}
+    bug_filed_24_hours_per_project = {}
+    valid_timestamp = datetime.datetime.now() - datetime.timedelta(hours=24)
+    bug_filing_max_24_hours_per_job = {}
+    bug_filing_max_24_hours_per_project = []
```

Comment:
Should this be a dictionary where the project name is the key? 

The project comes from https://github.com/google/clusterfuzz/blob/992274ebd11e178b2c664ec2fa684ff7d98df376/src/clusterfuzz/_internal/datastore/data_handler.py#L688. Note this is **different** from the issue tracker project name, which is used to extract the max thresholds in our config.